cost: 7
schedule:
  robot0:
  - t: 0
    x: 8.0
    y: 9.0
  - t: 1
    x: 8.0
    y: 10.0
  robot1:
  - t: 0
    x: 10.0
    y: 12.0
  robot2:
  - t: 0
    x: 11.0
    y: 11.0
  - t: 1
    x: 11.0
    y: 10.0
  - t: 2
    x: 11.0
    y: 9.0
  - t: 3
    x: 10.0
    y: 9.0
